examples/sample-deployments/kong-db/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../katalog/kong/kong-db
+
+patchesStrategicMerge:
+  - service.yaml
```

Comment:
Add a new line at the end of every file please

Reply:
👍